### PR TITLE
chore: add chaos-mesh to repositories.toml

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

#### ℹ️ Repository information

[Chaos Mesh](https://github.com/chaos-mesh/chaos-mesh) is a Cloud Native Chaos Engineering Platform for Kubernetes.

**The repository has**:

- [x] At least three issues with the good first issue label.
- [x] Detailed setup instructions for the project.
- [x] CONTRIBUTING.md
- [x] Actively maintained.

### CODE REVIEW

File: data/repositories.toml [hunk 1]
```diff
@@ -102,6 +102,7 @@ repositories = [
   'github.com/certbot/certbot',
   'github.com/CesiumGS/cesium',
   'github.com/chapel-lang/chapel',
+  'github.com/chaos-mesh/chaos-mesh',
   'github.com/chatwoot/chatwoot',
```
Comment (shubhendra-deepsource) [inline data/repositories.toml:106]:
```suggestion
  'github.com/CesiumGS/cesium',
  'github.com/chaos-mesh/chaos-mesh',
  'github.com/chapel-lang/chapel',
  'github.com/chatwoot/chatwoot',
```

**Review by shubhendra-deepsource — CHANGES_REQUESTED** [review]
@STRRL You missed it by one line. Make the change and we're good to go! 🚀

**Comment by STRRL** [comment]
> @STRRL You missed it by one line. Make the change and we're good to go!

updated! PTAL! XD

**Review by shubhendra-deepsource — APPROVED** [review]
LGTM

**Pull request merged**